Installing Q# for other IDEs (#911)
MicrosoftDocs/quantum-docs-pr: articles/install-guide/standalone.md

1. Download and install [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3 or greater, with the .NET Core cross-platform development workload enabled.
2. Download and install the [Microsoft QDK](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit).

To install the QDK for another environment, enter in the command line:

>     dotnet new -i Microsoft.Quantum.ProjectTemplates

Author: @geduardo @cgranade @tcNickolas I'm getting some very odd behavior here. This line was giving me wild rendering problems when code blocked with triple ``'s, so I went to make it in-line with single quotes. Then, it renders as a full code block, but with the single quotes included. I remove the quotes entirely, and voila, it's rendering perfectly as a code block without any instructions telling it to do so. 

Is there some trick (or bug) in the markdown scheme that I'm missing here?

Reviewer: I'm not sure why the ` ``` ` fences were failing, but indenting by four or more whitespace characters is an equivalent way of denoting literal code blocks, but comes with the disadvantage that you can't specify a language that way. The [DocFX Flavored Markdown reference](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html?tabs=tabid-1%2Ctabid-a) has more detail.

(As a side note, you can now include cells from Jupyter Notebook files...?)

>     `dotnet new -i Microsoft.Quantum.ProjectTemplates`

Author: ```suggestion
    dotnet new -i Microsoft.Quantum.ProjectTemplates
```

> [!NOTE]
> If you have multiple projects within one Visual Studio solution, all projects contained in the solution need to be in the same folder as the solution, or in one of its sub-folders.  

> ### [Command line or other editors](#tab/tabid-cmdline)

Reviewer 2: ```suggestion
### [Other editors with the command line](#tab/tabid-cmdline)
```

Verify your installation by creating a Q# `Hello World` application.

1. Create a new application:
    ```dotnetcli
    dotnet new console -lang Q# -o runSayHello
    ```

2. Navigate to the application directory:
    ```dotnetcli
    cd runSayHello
    ```

>     This directory should now contain a file `Program.qs`, which is a Q# program that defines a simple operation to print a message to the console.
> 

Reviewer 2: ```suggestion
    This directory should now contain a file `Program.qs`, which is a Q# program that defines a simple operation to print a message to the console. You can modfiy this template with a text editor and overwrite it with your own quantum applications. 

```
